semantic error: int.bit_length() takes no arguments
 --> tests/errors/test_bit_length.py:4:11
  |
4 |     print(x.bit_length(23))
  |           ^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
